README.md

```diff
@@ -172,6 +172,24 @@ Example
 </typeMapping>
 ```
 
+## typeMappingPropertiesFiles
+Provide typeMapping as properties file(s) that is accessible as a compile-time-classpath resource
+Key-Value pairs in the properties file will be added to `typeMapping` Map when it is not already present in it
+
+When a same GraphQL type is present in both `typeMapping` and also in `typeMappingPropertiesFiles`, value in `typeMapping` will be used (and the value from `typeMappingPropertiesFiles` *will* be ignored)
+
+- Type: Array
+- Required: false
+
+Example
+
+```xml
+<typeMappingPropertiesFiles>
+    <typeMappingPropertiesFile>commontypes-typeMapping.properties</typeMappingPropertiesFile>
```

Comment:
can I use filename patterns?

Reply:
Thanks @gkesler for reviewing this.
This PR expects mapping file to be in the classpath.  This can be enhanced to look from a file system too, but is that required?

Reply:
@gkesler did you mean regex style patterns to find properties files in the artifacts listed as dependencies for the project?